docs/content/metrics.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Metrics
+---
+
+# Prometheus Metrics
+
+## forwarder run
+
+### `forwarder_dialer_closed_total`
+
+Number of closed connections
+
+Labels:
+  - host
+### `forwarder_dialer_dialed_total`
```

Comment:
Can we have an empty line between labels and new metric. It does not matter in the final md output, yet it just feels better.